packages/fiori/src/Wizard.ts

```diff
@@ -251,7 +176,100 @@ const metadata = {
  * @appenddocs sap.ui.webc.fiori.WizardStep
  * @public
  */
+@customElement("ui5-wizard")
+@languageAware
+@fastNavigation
+
+/**
+ * Fired when the step is changed by user interaction - either with scrolling,
+ * or by clicking on the steps within the component header.
+ *
+ * @event sap.ui.webc.fiori.Wizard#step-change
+ * @param {WizardStep} step The new step.
+ * @param {WizardStep} previousStep The previous step.
+ * @param {boolean} changeWithClick The step change occurs due to user's click or 'Enter'/'Space' key press on step within the navigation.
+ * @public
+ */
+@event("step-change", {
+	detail: {
+		step: { type: WizardStep },
```

Comment:
we should use types only in TS related things. Here we describe step as HTMLElement.